patchouli_books/modded_for_dummies/en_us/entries/jei/jei_jer.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
we need to remove this, no JER for E9 :)

Reply:
yeeted